### PR TITLE
added landind msg #1

### PR DESCRIPTION
**Pull request by RomainCrt**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,6 +6,6 @@
 «title>Welcome to Potionfy</title>
 </head>
 <body>
-  
+  <h1>Welcome !!!!</h1>
```
Comment (basakbuluttekin) [inline index.html:9]: Too short.

Comment (RomainCrt) [reply index.html:9]: ok

Comment (basakbuluttekin) [inline index.html:9]: can you please have a look?

**Review by basakbuluttekin — CHANGES_REQUESTED** [review]

**Review by basakbuluttekin — APPROVED** [review]

**Comment by basakbuluttekin** [comment]
Well done!

**Pull request merged**